### PR TITLE
Add 'Use File Scoped Namespaces'

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Add 'Use File Scoped Namespaces' to the code-samples article in the cotribute folder as suggested in https://github.com/umbraco/UmbracoDocs/issues/5260#issuecomment-1647487697

This PR belongs to https://github.com/umbraco/UmbracoDocs/issues/5260

## Type of suggestion

* [x] Update contribution guidelines

### CODE REVIEW

File: generic/contribute/code-samples.md [hunk 1]
```diff
@@ -72,6 +72,28 @@ When reading any piece of text, there is nothing more frustrating than running i
 
 Any code that is added to articles in the documentation should be double-checked for syntax errors and typos.
 
+### Use File Scoped Namespaces
+The use of file scoped namespaces improves, among other things, the readability of the code samples. Therefore, use file scoped namespaces in the examples. See below how to use file scoped namespaces:
```
Comment (sofietoft) [inline generic/contribute/code-samples.md:76]: Just a few small adjustments:

```suggestion
The use of file-scoped namespaces improves, among other things, the readability of the code samples. Therefore, use file scoped namespaces in the examples. See below how to use file-scoped namespaces:
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Thanks for the PR @erikjanwestendorp ! 💪 

Looks great! I've added a single comment - take a look when you have a sec.

**Comment by erikjanwestendorp** [comment]
@sofietoft Thanks for your review, I just updated the PR with your suggestion.

**Comment by sofietoft** [comment]
Thanks @erikjanwestendorp ! 💪 

I'll get this merged!

**Pull request merged**